test/websocket.misc.js

```diff
@@ -0,0 +1,37 @@
+var assert = require('assert');
+var utils = require('./helpers/test.utils');
+var Web3 = utils.getWeb3();
+
+describe('Websocket etc & errors', function () {
```

Comment:
The description of this test file is to open and the file name itself as well. Is it possible to have a more describing file name and test description? If this isn't possible do we have to move those cases to somewhere else or we have to create additional smaller test files with more explicit file names and test descriptions.

Reply:
Sorry, yes the name is bad. I've made it more generic now (not sure if that's better) - if you have a good name llmk.

Reply:
I would move those test cases to the ``websocket.reconnect.ganache`` test file and rename it to ``e2e.websocket.js`` and let us create a real unit test withing the ``websocket.js`` test file.

Reply:
In 4044687b2992782eb2dffea26014a7f9ca56ef75 I combined the files and renamed it `websocket.ganche.js`. The files with the `e2e` prefix are grepped by mocha so that all the clients and browsers run them. 

But many of these tests rely on turning the server on and off, so they fit better with the units, idk. Similar to `eth.subscribe.ganache.js`

Reply:
Sounds good to me! We can probably improve "the e2e situation" later! :)